src/rsz/src/RepairDesign.cc

```diff
@@ -383,6 +624,21 @@ void RepairDesign::repairNet(Net* net,
                sdc_network_->pathName(drvr_pin));
     const Corner* corner = sta_->cmdCorner();
     bool repaired_net = false;
+
+    if (buffer_gain_ != 0.0) {
+      float fanout, max_fanout, fanout_slack;
+      // TODO: is it correct to unconditionally use max_fanout? what's fanout_slack?
+      sta_->checkFanout(drvr_pin, max_, fanout, max_fanout, fanout_slack);
+
+      int resized = resizer_->resizeToTargetSlew(drvr_pin);
```

Comment:
Is this needed?  abc won't do this.

Reply:
It’s not needed, it should be better if we do it than if we don’t (we can save on buffers if we can resize instead)

Reply:
The gain based buffering should be doing the sizing rather than relying on the target slew resizing (it tends to oversize and we are working to reduce/eliminate it).

Reply:
I thought they were a single algorithm.  In that case I guess you need to leave it until we have a better solution for sizing.  How is it in the current yosys/abc flow in terms of sizing and buffering?  Are they both run? In which order?

Reply:
> How is it in the current yosys/abc flow in terms of sizing and buffering? Are they both run? In which order?

Buffering then sizing

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/scripts/abc_speed.script

Reply:
So this would be sizing then buffering?

Reply:
Sizing, buffering, then sizing again, but on each driver at a time. So we don't do a separate pass of buffering and a separate pass of sizing where the buffering sees unsized loads or the other way around

Reply:
> Sizing, buffering, then sizing again

Now that I think of it there may be an interaction between buffering and sizing which explains why the combined process isn't idempotent, which I considered an outstanding issue with the code